src/components/AppSidebar/AppSidebar.vue

```diff
@@ -517,16 +517,15 @@ $desc-height: 46px;
 				}
 			}
 			// titles
-			h3, h4 {
+			h2, p.app-sidebar-header__subtitle {
```

Comment:
```suggestion
			h2,
			.app-sidebar-header__subtitle {
```
Best only use the class, just in case the HTML element ever changes